plotter.go

```diff
@@ -28,22 +29,22 @@ func GetPlotExpr(alertFormula string) []PlotExpr {
 	expr, _ := promql.ParseExpr(alertFormula)
 	if parenExpr, ok := expr.(*promql.ParenExpr); ok {
 		expr = parenExpr.Expr
-		log.Printf("Removing redundant brackets: %v", expr.String())
+		clog.Infof("Removing redundant brackets: %v", expr.String())
 	}
 
 	if binaryExpr, ok := expr.(*promql.BinaryExpr); ok {
```

Comment:
This code isn't useful to us at all.

It basically splits up joined expressions and evaluates each of them seperately and plots charts for each set of results. This normally leads to chunky alerts with one or both charts broken.

We should rework it so that it only evaluates the first of the joined expressions as we typically only use an AND when we need to validate results. Eg. (1) is blackbox probe successfully _and_ (2) are there other metrics for this host in other jobs - in this example we only care about charting (1) because if (2) did not meet threshold the alert wouldn't fire.

I'll log a ticket for this as it's out of scope.